src/python/pants/python/pex_build_util_test_integration.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import json
+import os
+import subprocess
+
+from pex.interpreter import PythonInterpreter
+
+from pants.testutil.pants_run_integration_test import PantsRunIntegrationTest
+from pants.util.collections import assert_single_element
+from pants.util.contextutil import open_zip, temporary_dir
+
+
+class PexBuildUtilIntegrationTest(PantsRunIntegrationTest):
+
+    binary_target_address = "testprojects/src/python/python_targets:test"
+
+    def test_multiplatform_python_binary(self):
```

Comment:
Nit: it's good to put `-> None` on tests. Why? Then MyPy will check the implementation of the test, whereas currently, MyPy doesn't go into this function's implementation at all.

Reply:
Thank you!!